File: scala-chapter-3/src/main/scala/org/tlh/springboot/scala/repository/UserRepository.scala
package org.tlh.springboot.scala.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.tlh.springboot.scala.entity.User

/**
  * <br>
  * Created by hu ping on 6/28/2019
  * <p>
  */
trait UserRepository extends JpaRepository[User, Int] {

}

File: proguard-chapter-1/src/main/java/org/tlh/springboot/repository/UserRepository.java
package org.tlh.springboot.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.tlh.springboot.entity.User;

/**
 * <br>
 * Created by hu ping on 8/15/2019
 * <p>
 */
public interface UserRepository extends JpaRepository<User, Integer> {
}
